src/discount.jl

```diff
@@ -11,25 +11,42 @@ end
 
 Discounter(rate, ts) = Discounter(rate, 1.0, ts)
 
-_start_strat(t::TimePeriod, ts::TimeStructure{T}) where {T} = zero(T)
+_start_strat(sp::AbstractStrategicPeriod, ts::TimeStructure{T}) where {T} = zero(T)
 
-function _start_strat(t::OperationalPeriod, ts::TwoLevel{S,T}) where {S,T}
-    sp = _strat_per(t)
-    if sp == 1
-        return zero(S)
-    end
+function _start_strat(sp::AbstractStrategicPeriod, ts::TwoLevel{S}) where {S}
+    return sum(duration_strat(spp) for spp in strat_periods(ts) if spp < sp; init = zero(S))
+end
 
-    return sum(duration_strat(spp) for spp in strat_periods(ts) if _strat_per(spp) < sp)
+function _start_strat(sp::StratNode, ts::TwoLevelTree{S}) where {S}
+    start = zero(S)
+    node = sp
```

Comment:
```suggestion
    node = _parent(sp)
```
Otherwise, we also include the current strategic period. At the time being, it is not equivalent to the version dispatching on `AbstractStrategicPeriod`.